submodules/privatelink/vpc-endpoint-services.tf

```diff
@@ -0,0 +1,153 @@
+locals {
+  endpoint_services = tomap({ for service in var.vpc_endpoint_services : service.name => service.private_dns })
```

Comment:
is this `tomap` needed?

Reply:
making it a map it is easier to work in the tf

Reply:
any suggestion¿

Reply:
Oh I just wasn't sure why the tomap was needed I thought the map was created with
```
endpoint_services = { for service in var.vpc_endpoint_services : service.name => service.private_dns }
```

Reply:
let me check